app/presenters/brexit_landing_page_presenter.rb

```diff
@@ -31,6 +31,10 @@ def supergroup_sections
     end
   end
 
+  def email_path
+    base_path.gsub(/(\.\w{2})$/, "")
```

Comment:
Minor: we could do `.split('.').first` here.

Reply:
Done 👍